install.sh

```diff
@@ -0,0 +1,448 @@
+#!/usr/bin/env bash
+# modifed from https://github.com/v2fly/fhs-install-v2ray/blob/master/install-release.sh
+# You can set this variable whatever you want in shell session right before running this script by issuing:
+# export JSON_PATH='/usr/local/etc/hysteria'
+JSON_PATH=${JSON_PATH:-/etc/hysteria}
+
+curl() {
+  $(type -P curl) -L -q --retry 5 --retry-delay 10 --retry-max-time 60 "$@"
```

Comment:
Tip: you can use `command` built-in like `command curl -L -q --retry ...` to force bash to execute external curl program.